app/components/elements/image_component.rb

```diff
@@ -22,8 +23,15 @@ def setup_classes
       add_classes('d-none d-md-block') if @collapse
     end
 
+    def style
+      style = ''
+      style += "width: #{@width};" if @width
+      style += " height: #{@height};" if @height
```

Comment:
Didn't end up using these changes but left them here in case they are needed later.